easybuild/easyconfigs/m/MariaDB-connector-c/MariaDB-connector-c-2.3.4-intel-2017b.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'CMakeMake'
+
+name = 'MariaDB-connector-c'
+version = '2.3.4'
+
+homepage = 'https://downloads.mariadb.org/connector-c/'
+description = "MariaDB Connector/C is used to connect applications developed in C/C++ to MariaDB and MySQL databases."
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+source_urls = ['https://downloads.mariadb.org/f/connector-c-%(version)s']
+sources = ['mariadb-connector-c-%(version)s-src.tar.gz']
+checksums = ['8beb0513da8a24ed2cb47836564c8b57045c3b36f933362f74b3676567c13abc']
+
+builddependencies = [('CMake', '3.10.1')]
+dependencies = []
```

Comment:
Is this necessary?

Reply:
Hah, of course not, I was expected that a full blown `MariaDB` was going to be required here, but that's not the case. Will clean up, thanks